README.md

```diff
@@ -3,40 +3,114 @@
 
 ## Architecutre
 
-The policy framework consists of following components:
-
-- [Governance dashboard](https://github.com/open-cluster-management/grc-ui)
-- [Policy propagator](https://github.com/open-cluster-management/governance-policy-propagator) 
-- [Policy spec sync](https://github.com/open-cluster-management/governance-policy-spec-sync)
-- [Policy status sync](https://github.com/open-cluster-management/governance-policy-status-sync)
-- [Policy template sync](https://github.com/open-cluster-management/governance-policy-template-sync)
-- Policy controllers: Policy controllers include predefined [out-of-box policy controllers](#out-of-box-policies-and-controllers), or you can [bring your own policy](#bring-your-own-policy).
-
 ![architecture](images/policy-framework-architecture-diagram.jpg)
 
-## Out-of-box policies and controllers
-
-View the following list of predefined policy controllers that are offered with Red Hat Advanced Cluster Management:
-
-- [Configuration policy controller](https://github.com/open-cluster-management/config-policy-controller)
-- [Certificate policy controller](https://github.com/open-cluster-management/cert-policy-controller)
-- [IAM policy controller](https://github.com/open-cluster-management/iam-policy-controller)
-
-## Bring your own policy
+The governance policy framework consists of following components:
+
+- [Governance dashboard](https://github.com/open-cluster-management/grc-ui) -- UI
+- Govenance policy framework -- A framework to distribute various supported policies to managed cluster and collect results back to hub. It consists of following components:
+    - [Policy propagator](https://github.com/open-cluster-management/governance-policy-propagator) 
+    - [Policy spec sync](https://github.com/open-cluster-management/governance-policy-spec-sync)
+    - [Policy status sync](https://github.com/open-cluster-management/governance-policy-status-sync)
+    - [Policy template sync](https://github.com/open-cluster-management/governance-policy-template-sync)
+- Policy controllers -- Policy engines running on managed cluster to evaluate policy rules distributed by the policy framework and generate results.
+    - [Configuration policy controller](https://github.com/open-cluster-management/config-policy-controller)
+    - [Certificate policy controller](https://github.com/open-cluster-management/cert-policy-controller)
+    - [IAM policy controller](https://github.com/open-cluster-management/iam-policy-controller)
+    - 3rd party
+      - [Gatekeeper](https://github.com/open-policy-agent/gatekeeper)
+      - [Kyverno](https://github.com/kyverno/kyverno/)
+      - [Bring your own](#bring-your-own-policy-controller)
+
+## The Policy CRDs
+
+The `Policy` is the Custom Resource Definition (CRD), created for policy framework controllers to monitor. It acts as a vechical to deliver policies to managed cluster and collect results back to hub.
+This is an example spec of a `Policy` object:
+```yaml
```

Comment:
I recommend that you update "This is an example spec of a `Policy` object:" 

TO 

"View the following example specification of a `Policy` object"